plugins/mgcp.js

```diff
@@ -1237,6 +1237,24 @@ mgcp = {
                 delete tags.place;
                 break;
 
+            case 'island':
+            case 'islet':
+                // If we have a coastline around an Island, decide if we are going make an Island
+                // or a Coastline
+                if (tags.natural == 'coastline')
```

Comment:
The lack of curly braces following this if statement freaks me out. It's probably not strictly required, but I think good style would dictate their addition.